src/utils/date.ts

```diff
@@ -0,0 +1,7 @@
+export function formatDate(date: Date): string {
```

Comment:
Format date as what? This isn't clear what it formats as. It also doesn't use our temporal types